modules/ocf_printhost/files/enforcer

```diff
@@ -167,12 +185,23 @@ def send_printer_mail(message, job, quo):
         semester_pages=quo.semesterly,
         daily_quota=quota.daily_quota(),
         semester_quota=quota.SEMESTERLY_QUOTA,
+        id=job.id
     )
     send_mail_user(job.user, message.subject, body)
 
 
+def statusUpdate(r, job, statusCode):
```

Comment:
not really relevant or needed, but maybe `statusUpdate` -> `status_update`/`publish_status` to keep in line with style